.github/workflows/ci-backend.yml

```diff
@@ -153,14 +153,6 @@ jobs:
                   path: 'posthog-cloud-infra'
                   token: ${{ secrets.POSTHOG_BOT_GITHUB_TOKEN }}
```

Comment:
@aspicer Alright if we remove this `actions/checkout@v4` step too? Causing a red check on #25265